frontend/src/widgets/HorizontalTableWidget.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table } from '@trussworks/react-uswds';
+import Container from '../components/Container';
+import './HorizontalTableWidget.scss';
+
+const trimLongURLs = (url) => {
+  let newUrl = url;
+  if (newUrl.length >= 40) {
+    newUrl = newUrl.substring(0, 40);
+    newUrl += '...';
+  }
+  return newUrl;
+};
+
+export default function HorizontalTableWidget(
+  {
+    title,
+    subtitle,
+    headers,
+    data,
+    loading,
+    loadingLabel,
+    firstHeading,
+    lastHeading,
+  },
+) {
+  return (
+    <Container className="smarthub-horizontal-table-widget shadow-2 padding-top-0" paddingX={0} paddingY={0} loading={loading} loadingLabel={loadingLabel}>
+      <div className="margin-bottom-1 padding-top-3 padding-left-3 margin-bottom-3">
+        <h2 className="smart-hub--table-widget-heading margin-0 font-sans-lg">{title}</h2>
+        <p className="usa-prose margin-0">{subtitle}</p>
+      </div>
+      {/* a scrollable element must be keyboard accessible */}
+      {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+      <div className="usa-table-container--scrollable margin-top-0" tabIndex={0}>
+        <Table fullWidth striped bordered={false}>
+          <thead>
+            <tr>
+              <th className="smarthub-horizontal-table-first-column">
+                {firstHeading}
+              </th>
+              {headers.map((h) => <th key={h.replace(' ', '_')} scope="col" className="text-left">{h}</th>)}
+              <th className="smarthub-horizontal-table-last-column">
+                {lastHeading}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {
+            data.map((r, index) => (
+              <tr key={`horizontal_table_row_${index}`}>
+                <td key={`horizontal_table_cell_label${index}`} className="smarthub-horizontal-table-first-column">
+                  {
+                    r.isUrl === 'true'
+                      ? (
+                        <a style={{ display: 'table-cell' }} title="Links to Resource" aria-label={`Links to Resource ${r.heading}`} href={r.heading} target="_blank" rel="noreferrer">
```

Comment:
you can remove that aria label, it's not necessary

Reply:
also curious as to what display: table-cell is doing, I wouldn't think you need it. I removed it locally and didn't see a difference

also please remove that title attribute, bad practice to have those on there